targets/TARGET_NUVOTON/TARGET_NANO100/device/TOOLCHAIN_ARM_MICRO/NANO130.sct

```diff
@@ -1,27 +1,41 @@
 #! armcc -E
 
+; 512 KB APROM
+#if !defined(MBED_APP_START)
+  #define MBED_APP_START            0x00000000
+#endif
+
+#if !defined(MBED_APP_SIZE)
+  #define MBED_APP_SIZE             0x00020000
+#endif
+
+; 64 KB SRAM (internal)
+#define MBED_RAM_START              0x20000000
+#define MBED_RAM_SIZE               0x4000
+
 #if !defined(MBED_BOOT_STACK_SIZE)
   #define MBED_BOOT_STACK_SIZE      0x400
 #endif
 
-LR_IROM1 0x00000000 {
-  ER_IROM1 0x00000000 {  ; load address = execution address
+#define VECTOR_SIZE                 (4*(16 + 142))
```

Comment:
The interrupt number in `VECTOR_SIZE` macro definition is incorrect. Because **TARGET_NANO100** port doesn't support vector table relocation. Just remove the `VECTOR_SIZE` macro definition.